packages/engines/snakemake/buildspec.yml

```diff
@@ -0,0 +1,22 @@
+version: 0.2
+
+env:
+  shell: bash
+  variables:
+    # These variables may be over-ridden as appropriate by the CI/CD pipeline
+    SNAKEMAKE_IMAGE_NAME: "snakemake"
+
+phases:
+  pre_build:
+    commands:
+      - SNAKEMAKE_IMAGE_URI=${SNAKEMAKE_IMAGE_NAME}:latest
```

Comment:
We may want to pin to a version here - using latest may introduce unwanted bugs that are hard to reproduce

Reply:
good call out. Fixed in second commit.